help_to_heat/portal/download_views.py

```diff
@@ -77,12 +77,20 @@ def add_extra_row_data(referral):
     eligibility = calculate_eligibility(row)
```

Comment:
Now it's more like "update rows" instead of "add extra rows" if I have not misunderstood 🤔

Reply:
The method name is "add extra row *data*". It is adding extra data to the rows, not adding rows 🙂

Reply:
Ahhh yes of cause!! :comfy-panic: :facepalm: